gnss_lib_py/utils/ephemeris_downloader.py

```diff
@@ -760,7 +1019,7 @@ def _get_rinex_extension(timestamp):
 
     """
     # switched from .Z to .gz compression format on December 1st, 2020
```

Comment:
This comment does not quite match the updated code, though the docstring on line 1004 does match the code.

Reply:
removed this line comment in preference for the existing function docstring which has the correct information.